sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/TraceHelper.cs

```diff
@@ -168,5 +177,71 @@ internal static string GetOperationName(Activity activity, ref AzMonList MappedT
 
             return activity.DisplayName;
         }
+
+        private static void AddExceptionTelemetryFromActivityExceptionEvents(Activity activity, ref TagEnumerationState monitorTags, string roleName, string roleInstance, string instrumentationKey, List<TelemetryItem> telemetryItems)
+        {
+            if (activity.Events.Any())
+            {
+                foreach (var evnt in activity.Events)
+                {
+                    if (evnt.Name == SemanticConventions.AttributeExceptionEventName)
```

Comment:
can we exit the for loop once we fine the exception event? Do you want to support the case where user has added more than exception to events?

Reply:
Yes - I was considering the case where user has called `activity.RecordException(ex)` multiple times